basics/card_types.md

```diff
@@ -1,82 +1,58 @@
 # Card Types
 
+[//]: # 'Comment on card parts'
```

Comment:
What does this mean?

Reply:
On top of the card you can see the name. on the left side you can see some numbers indicating the cost of the card. These are inside circles that indicate what 'colour' of energy you need to spend on them. Below the art you can see the card type and subtypes, and at the bottom there's the text box
Or something like this